.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "files.associations": {
+        "chrono": "cpp"
+    }
+}
```

Comment:
@mked-luxoft Do we really need this file related to VSCode in repo?

Reply:
no we don't. it was `git add .` , sorry :)